### DISCUSSION TITLE
Change IP address

### DISCUSSION THREAD

**Discussion by jjm888** [tillsteinbach/VWsFriend#171] [Q&A] [answered]
Hi, great app now installed on one of my Pi's. Having moved to another Pi with a different IP address how do I rerun setup to  set IP address? Thanks

**Comment by tillsteinbach** [comment, marked as answer]
The path is `/database/settings/edit` I have to make that available through a link. My bad!

**Reply by tillsteinbach** [reply]
By the way: I guess you used the backup/restore feature. Did that go smoothly?

**Reply by jjm888** [reply]
Thanks fixed ::)
